src/session.rs

```diff
@@ -860,6 +861,28 @@ impl Session {
 
         Ok(())
     }
+
+    pub(crate) fn set_direction(&mut self, mid: Mid, direction: Direction) -> bool {
+        let Some(media) = self.media_by_mid_mut(mid) else {
+            return false;
```

Comment:
Will this ever happen? If it happens does it matter if it returns true or false?

Reply:
I guess if someone uses the SDP change to set a direction for a non-existing MID.

But could also be relevant if we expose this in the Direct API.

It could be an error.

Reply:
The documentation currently says:

```text
    /// If the direction is set for media that doesn't exist, or if the direction is
    /// the same that's already set [`SdpApi::apply()`] not require a negotiation.
```